CommandLine/testing/GitHub-ImageDiff.sh

```diff
@@ -41,8 +41,8 @@ master_dir="/tmp/enigma-master/test-harness-out"
 pr_dir="${PWD}/test-harness-out"
 diff_dir="/tmp"
 
-master_images=$(ls ${master_dir}/*.png | xargs basename | sort)
-pr_images=$(ls ${pr_dir}/*.png | xargs basename | sort)
+master_images=$(ls ${master_dir}/*.png | xargs -n1 basename | sort)
+pr_images=$(ls ${pr_dir}/*.png | xargs -n1 basename | sort)
 
 echo "${master_images}" > "/tmp/master_images.txt"
```

Comment:
You should be able to just call, eg,
```bash
com_master_pr=$(comm -2 -3 <(echo $master_images) <(echo $pr_images) | tr -d '\t')
```

Thereby obviating the need for temp files (actually, this just creates them for you as an "anonymous fifo").

Reply:
I'm trying out your process substitution now: 17c2a93ffcf59569921bbaefea1fc178574f2d59

Reply:
It didn't work the first time but I finally got it in: d29e6928a034f1cb7807b0a10a872de9ce34930a

The vars needed quoted because they have newlines as we discussed on Discord. That did fix it and then the bot commented with the correct feedback.